lib/geoengineer/resources/aws_api_gateway_api_key.rb

```diff
@@ -0,0 +1,18 @@
+########################################################################
+# AwsCloudTrail is the +aws_api_gateway_api_key+ terrform resource,
+#
+# {https://www.terraform.io/docs/providers/aws/r/aws_api_gateway_api_key.html}
+########################################################################
+# TODO: not fully implemented
```

Comment:
no, the resource is just a place holder for when we do get around to implementing it.